src/main/java/com/example/bookstore/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,28 @@
+package com.example.bookstore.exception;
+
+import java.util.List;
+import org.springframework.context.support.DefaultMessageSourceResolvable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<List<String>> handleValidationException(
+            MethodArgumentNotValidException e) {
+        final List<String> errors = e.getBindingResult().getAllErrors()
+                .stream()
+                .map(DefaultMessageSourceResolvable::getDefaultMessage)
+                .toList();
+        return new ResponseEntity<>(errors, HttpStatus.BAD_REQUEST);
+    }
+
+    @ExceptionHandler(Exception.class)
+    public ResponseEntity<String> handleAllExceptions(Exception e) {
+        return new ResponseEntity<>("An unexpected error occurred",
```

Comment:
It's recommended to use JSON format in REST applications for all responses. So create some error response dto with details fields.